swift-sdk/Internal/EmptyFlexMessagingManager.swift

```diff
@@ -0,0 +1,9 @@
+//
+//  Copyright © 2023 Iterable. All rights reserved.
+//
+
+import Foundation
+
+class EmptyFlexMessagingManager: IterableFlexMessagingManagerProtocol {
```

Comment:
Whats the purpose of having an empty manager? I noticed there is one for in-app manager too. Does it act as an interface?

Reply:
Should this be inheriting from IterableInternalFlexMessagingManagerProtocol to keep with the convention of the EmptyInAppManager?

Reply:
It's a convention when the SDK doesn't initialize correctly, [over here](https://github.com/Iterable/swift-sdk/blob/4720adacaa87249d92e8d873ea8381513ea46658/swift-sdk/IterableAPI.swift#L197).

There isn't a `IterableInternalFlexMessagingManagerProtocol`, I'm not sure it'll be necessary, but for now, no.